publications-2017.bib

```diff
@@ -1,5 +1,102 @@
 % Encoding: US-ASCII
 
+@PhDThesis{Cox2017,
+  Title                    = {{Adaptive large-scale mantle convection simulations}},
+  Author                   = {Cox, SP},
+  Year                     = {2017},
+  School = {University of Leicester},
+  Url                      = {https://lra.le.ac.uk/handle/2381/39571}
+}
+
+@Article{Ganesan2017,
+  Title                    = {{Microstructural Response of Magnesium Alloys: 3D Crystal Plasticity and Experimental Validation}},
+  Author                   = {Ganesan, S},
+  Year                     = {2017},
+  School = {University of Michigan},
+  Url                      = {http://www-personal.umich.edu/~veeras/papers/sriramthesis.pdf}
+}
+
+@PhdThesis{Giuliani2017,
+  Title                    = {{Modelling Fluid Structure Interaction problems using Boundary Element Method}},
+  Author                   = {Giuliani, N},
+  School                   = {Scuola Internazionale Superiore di Studi Avanzati - Trieste},
+  Year                     = {2017},
+  Url                      = {https://iris.sissa.it/handle/20.500.11767/57293}
+}
+
+@InProceedings{Goluchova,
+  Title                    = {Epicyclic oscillations of thick relativistic disks},
+  Author                   = {Goluchov{\'{a}}, K and T{\"{o}}r{\"{o}}k, G},
+  Booktitle                  = {Proceedings of RAGtime 17–19, 17–19/23–26 Oct., 1–5 Nov., 2015/2016/2017, Opava, Czech Republic},
+  Editor = {Z. Stuchl{\’i}k, G. Tor{\”o}k and V. Karas},
+  Pages = {47--59},
+  Url                      = {http://proceedings.physics.cz/images/proc17/hor2.pdf}
+}
+
+@article{Grohs2017,
+  doi = {10.5802/smai-jcm.26},
+  url = {https://doi.org/10.5802/smai-jcm.26},
+  year = {2017},
+  publisher = {Cellule {MathDoc}/{CEDRAM}},
+  volume = {3},
+  pages = {219--248},
+  author = {Philipp Grohs and Ralf Hiptmair and Simon Pintarelli},
```

Comment:
Should we also do `Grohs, P` etc.?

Reply:
It's the same to bibtex -- "Lastname, Firstname" and "Firstname Lastname" are the same.

Reply:
I phrased my question poorly. I really meant to ask if we should abbreviate the first names of the authors like we do elsewhere.